src/main/java/org/jscep/server/ScepServlet.java

```diff
@@ -623,7 +613,15 @@ protected abstract X509CRL doGetCrl(final X500Name issuer,
         if (req.getMethod().equals(POST)) {
             return IOUtils.toByteArray(req.getInputStream());
         } else {
-            Operation op = getOperation(req);
+            final Operation op;
```

Comment:
Can you explain the intent here?

Reply:
If someone passes a bogus operation (such that Operation.forName throws an IllegalArgumentException), the response code is a 500 because there's nothing catching the exception.  So I added a try/catch block here.  I thought about putting the call to getMessageBytes inside the existing try/catch block above, but somehow that seemed a little risky, as there could (at least in theory) be multiple reasons for IllegalArgumentException with multiple method calls inside the catch block.